### ISSUE TITLE
Display button icon and use button text for aria-label when both `buttonText` and `buttonIcons` options are set

### ISSUE THREAD

**Issue by goulvench** [fullcalendar/fullcalendar#6012]
### Checklist

  - [X] I've already searched through [existing tickets](https://fullcalendar.io/issues)
  - [X] Other people will find this feature useful

### Feature Description

Fullcalendar v5 allows toolbar buttons to contain either text or icons. However, when both options are set, `buttonText` takes precedence and no icons are displayed (this should be documented by the way, I found it the hard way after some head-scratching).

If both `buttonText` and `buttonIcons` are set, I suggest applying the buttonText as `aria-label` to the button, and applying the font-awesome icon normally. As a side-effect, this would greatly improve accessibility (the current aria-label for a `dayGridMonth` button is `dayGridMonth`, which is neither a word nor localized).

**[Edit]** Another option would be to display text and icon side-by-side, and introduce another option for setting `aria-label`s. 

### Backwards compatibility

This would be a breaking change, since users having set both options would now see an icon instead of text, but since both options are mutually exclusive I believe the gain would counterbalance the loss because most users have set either one or the other option.

### Code

Current HTML when using `buttonIcons`
```html
<div class="fc-button-group">
    <button class="fc-dayGridMonth-button fc-button fc-button-primary" type="button" aria-label="dayGridMonth">
        <span class="fc-icon fc-icon-calendar"></span>
    </button>
    <button class="fc-timeGridWeek-button fc-button fc-button-primary fc-button-active" type="button" aria-label="dayGridWeek">
        <span class="fc-icon fc-icon-calendar-week"></span>
    </button>
</div>
```

Current HTML when using `buttonText` OR both `buttonText` and `buttonIcons`
```html
<div class="fc-button-group">
    <button class="fc-dayGridMonth-button fc-button fc-button-primary" type="button">Month</button>
    <button class="fc-timeGridWeek-button fc-button fc-button-primary fc-button-active" type="button">Week</button>
</div>
```

Proposed HTML when using both `buttonText` and `buttonIcons`
```html
<div class="fc-button-group">
    <button class="fc-dayGridMonth-button fc-button fc-button-primary" type="button" aria-label="Month">
        <span class="fc-icon fc-icon-calendar"></span>
    </button>
    <button class="fc-timeGridWeek-button fc-button fc-button-primary fc-button-active" type="button" aria-label="Week">
        <span class="fc-icon fc-icon-calendar-week"></span>
    </button>
</div>
```

**[Edit]** Proposed HTML when using `buttonText`, `buttonIcons`, and a new option `buttonLabel`
```html
<div class="fc-button-group">
    <button class="fc-dayGridMonth-button fc-button fc-button-primary" type="button" aria-label="Month">
        <span class="fc-icon fc-icon-calendar"></span>Month
    </button>
    <button class="fc-timeGridWeek-button fc-button fc-button-primary fc-button-active" type="button" aria-label="Week">
        <span class="fc-icon fc-icon-calendar-week"></span>Week
    </button>
</div>

```

**Comment by arshaw** [comment]
@goulvench , thank you for your thorough ticket. It helped me think about this problem. I have created a new ticket to supersede this one:
https://github.com/fullcalendar/fullcalendar/issues/6521
&uarr;
Could you please visit the link, **give a** 👍  **to vote** ([?](https://github.com/blog/2119-add-reactions-to-pull-requests-issues-and-comments)), press the 🔈**Subscribe** button to receive updates?
